.well-known/assetlinks.json

```diff
@@ -0,0 +1,8 @@
+[{
+    "relation": ["delegate_permission/common.handle_all_urls"],
+    "target": {
+      "namespace": "android_app",
+      "package_name": "com.expensify.chat",
+      "sha256_cert_fingerprints": ["2E:65:6F:1C:34:F5:7E:BF:FC:C0:2D:A3:14:0E:83:FE:61:51:F2:9B:5D:59:58:61:C4:4D:A9:99:0C:CA:F4:8E"]
```

Comment:
Is this correct? Or we have to change it.

Reply:
I extracted it from the New Expensify's APK in the Play Store, so it should probably work.

Reply:
@AndrewGable Could you please verify this? https://developer.android.com/training/app-links/verify-site-associations#web-assoc .

Reply:
@AndrewGable might be a bit slow to respond this week… He's getting married 😅 💍

Reply:
Awesome, definitely he won't be able to respond. In that case, @deetergp Could you please check the signature? https://developer.android.com/training/app-links/verify-site-associations#web-assoc has details to get the signature.

Reply:
Yes.. Please let's double check this..

Reply:
Verified in Google Play store and it is correct!
![Screen Shot 2022-08-03 at 1 02 00 PM](https://user-images.githubusercontent.com/22219519/182688498-9cc08669-f706-4d06-8a73-5601fd5c3b73.png)

Reply:
Thanks @luacmartins